Upgrading to Kotlin 1.3.30 causes "A failure occurred while executing org.jmailen.gradle.kotlinter.tasks.lint.LintWorkerRunnable  > PROPERTY_DELEGATE"

When I upgrade from Kotlin 1.3.21 to 1.3.30 I get the followning when execute the gradle `check` task:

```
> Task :lintKotlinMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lintKotlinMain'.
> There was a failure while executing work items
   > A failure occurred while executing org.jmailen.gradle.kotlinter.tasks.lint.LintWorkerRunnable
      > PROPERTY_DELEGATE
```

This is using kotlinter 1.23.1.

If I don't apply the kotlinter plugin but keep everything else the same, the `check` task passes.

Here's the stack trace:

```
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':lintKotlinMain'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
        at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:51)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:301)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:293)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:175)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:46)
        at org.gradle.execution.taskgraph.LocalTaskInfoExecutor.execute(LocalTaskInfoExecutor.java:42)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:277)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:262)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:135)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:130)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.execute(DefaultTaskPlanExecutor.java:200)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.executeWithWork(DefaultTaskPlanExecutor.java:191)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.run(DefaultTaskPlanExecutor.java:130)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: org.gradle.workers.WorkerExecutionException: There was a failure while executing work items
        at org.gradle.workers.internal.DefaultWorkerExecutor.workerExecutionException(DefaultWorkerExecutor.java:155)
        at org.gradle.workers.internal.DefaultWorkerExecutor.await(DefaultWorkerExecutor.java:149)
        at org.jmailen.gradle.kotlinter.tasks.LintTask.run(LintTask.kt:84)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:801)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:768)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:131)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:301)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:293)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:175)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:120)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:99)
        ... 31 more
Caused by: org.gradle.workers.internal.DefaultWorkerExecutor$WorkExecutionException: A failure occurred while executing org.jmailen.gradle.kotlinter.tasks.lint.LintWorkerRunnable
        at org.gradle.workers.internal.DefaultWorkerExecutor$WorkerExecution.waitForCompletion(DefaultWorkerExecutor.java:274)
        at org.gradle.internal.work.DefaultAsyncWorkTracker.waitForItemsAndGatherFailures(DefaultAsyncWorkTracker.java:115)
        at org.gradle.internal.work.DefaultAsyncWorkTracker.waitForCompletion(DefaultAsyncWorkTracker.java:87)
        at org.gradle.workers.internal.DefaultWorkerExecutor.await(DefaultWorkerExecutor.java:147)
        ... 46 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.github.shyiko.ktlint.core.ast.ElementType
        at com.github.shyiko.ktlint.ruleset.standard.ChainWrappingRule.<init>(ChainWrappingRule.kt:36)
        at com.github.shyiko.ktlint.ruleset.standard.StandardRuleSetProvider.get(StandardRuleSetProvider.kt:12)
        at org.jmailen.gradle.kotlinter.support.RuleSetsKt.resolveRuleSets(ruleSets.kt:15)
        at org.jmailen.gradle.kotlinter.support.RuleSetsKt.resolveRuleSets$default(ruleSets.kt:11)
        at org.jmailen.gradle.kotlinter.tasks.lint.LintWorkerRunnable.run(LintWorkerRunnable.kt:47)
        at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:39)
        at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:25)
        at org.gradle.workers.internal.NoIsolationWorkerFactory$1$1.call(NoIsolationWorkerFactory.java:69)
        at org.gradle.workers.internal.NoIsolationWorkerFactory$1$1.call(NoIsolationWorkerFactory.java:63)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:315)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:305)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:175)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:101)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
        at org.gradle.workers.internal.NoIsolationWorkerFactory$1.execute(NoIsolationWorkerFactory.java:63)
        at org.gradle.workers.internal.DefaultWorkerExecutor$1.call(DefaultWorkerExecutor.java:102)
        at org.gradle.workers.internal.DefaultWorkerExecutor$1.call(DefaultWorkerExecutor.java:96)
        at org.gradle.internal.work.AbstractConditionalExecution$1.run(AbstractConditionalExecution.java:38)
        at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.runExecution(DefaultConditionalExecutionQueue.java:212)
        at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.runBatch(DefaultConditionalExecutionQueue.java:161)
        at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.run(DefaultConditionalExecutionQueue.java:130)
        ... 3 more
```

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@ Available on the Gradle Plugins Portal: https://plugins.gradle.org/plugin/org.jm
 
 ```kotlin
 plugins {
-    id("org.jmailen.kotlinter") version "1.23.1"
+    id("org.jmailen.kotlinter") version "1.24.0"
 }
 ```
 
@@ -26,7 +26,7 @@ plugins {
 
 ```groovy
 plugins {
-    id "org.jmailen.kotlinter" version "1.23.1"
+    id "org.jmailen.kotlinter" version "1.24.0"
 }
 ```
 
@@ -46,7 +46,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath("org.jmailen.gradle:kotlinter-gradle:1.23.1")
+        classpath("org.jmailen.gradle:kotlinter-gradle:1.24.0")
     }
 }
 ```
@@ -71,7 +71,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath "org.jmailen.gradle:kotlinter-gradle:1.23.1"
+        classpath "org.jmailen.gradle:kotlinter-gradle:1.24.0"
     }
 }
 ```
@@ -85,6 +85,8 @@ apply plugin: "org.jmailen.kotlinter"
 </details>
 
 ### Compatibility
+Kotlinter 1.24.0 and later compatible with Kotlin Gradle plugins 1.3.30+ and Java 11/10/9/8.
+
 Kotlinter 1.21.0 and later compatible with Kotlin Gradle plugins 1.3.20+ and Java 11/10/9/8.
 
 Kotlinter 1.12.0 and later compatible with Kotlin Gradle plugins 1.2.41+ and Java 9/8.
@@ -181,7 +183,7 @@ kotlinter {
     ignoreFailures = false
     indentSize = 4
     continuationIndentSize = 4
-    reporter = arrayOf("checkstyle", "plain")
+    reporters = arrayOf("checkstyle", "plain")
     experimentalRules = false
     fileBatchSize = 30
 }
@@ -209,7 +211,7 @@ Options for `reporters`: checkstyle, html, json, plain
 
 The html reporter is provided by [ktlint-html-reporter](https://github.com/mcassiano/ktlint-html-reporter).
 
-Reporters behave as described at: https://github.com/shyiko/ktlint
+Reporters behave as described at: https://github.com/pinterest/ktlint
 
 *Note: `reporter` with a single value is deprecated but supported for backwards compatibility.
 
@@ -258,7 +260,7 @@ If you need to use a different version of `ktlint` you can override the dependen
 ```kotlin
 buildscript {
     configurations.classpath
-        .resolutionStrategy.force("com.github.shyiko:ktlint:0.28.0")
+        .resolutionStrategy.force("com.github.pinterest:ktlint:0.31.0")
 }
 ```
 
@@ -270,7 +272,7 @@ buildscript {
 ```groovy
 buildscript {
     configurations.classpath {
-        resolutionStrategy { force 'com.github.shyiko:ktlint:0.28.0' }
+        resolutionStrategy { force 'com.github.pinterest:ktlint:0.31.0' }
     }
 }
 ```

--- a/build.gradle
+++ b/build.gradle
@@ -1,9 +1,9 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.21'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.31'
     id 'com.gradle.plugin-publish' version '0.10.0'
     id 'java-gradle-plugin'
     id 'maven-publish'
-    id 'org.jmailen.kotlinter' version '1.22.0'
+    // id 'org.jmailen.kotlinter' version '1.22.0'
     id 'idea'
 }
 
@@ -14,7 +14,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.github.shyiko:ktlint:0.31.0'
+    implementation 'com.pinterest:ktlint:0.32.0'
     implementation 'me.cassiano:ktlint-html-reporter:0.2.0'
     
     compileOnly 'org.jetbrains.kotlin:kotlin-gradle-plugin'
@@ -30,7 +30,7 @@ tasks.withType(PluginUnderTestMetadata).configureEach {
     pluginClasspath.from(configurations.compileOnly)
 }
 
-version = '1.23.1'
+version = '1.24.0'
 group = 'org.jmailen.gradle'
 def pluginId = 'org.jmailen.kotlinter'
 
@@ -66,5 +66,5 @@ publishing {
 }
 
 wrapper {
-    gradleVersion = '5.3.1'
+    gradleVersion = '5.4.1'
 }

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/support/ExecutionContext.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/support/ExecutionContext.kt
@@ -1,6 +1,6 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.Reporter
+import com.pinterest.ktlint.core.Reporter
 import org.gradle.api.logging.Logger
 
 /**

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/support/HasErrorReporter.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/support/HasErrorReporter.kt
@@ -1,7 +1,7 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.LintError
-import com.github.shyiko.ktlint.core.Reporter
+import com.pinterest.ktlint.core.LintError
+import com.pinterest.ktlint.core.Reporter
 import java.util.concurrent.atomic.AtomicBoolean
 
 /**

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/support/SortedThreadSafeReporterWrapper.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/support/SortedThreadSafeReporterWrapper.kt
@@ -1,7 +1,7 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.LintError
-import com.github.shyiko.ktlint.core.Reporter
+import com.pinterest.ktlint.core.LintError
+import com.pinterest.ktlint.core.Reporter
 import java.util.concurrent.ConcurrentHashMap
 import java.util.concurrent.ConcurrentMap
 import java.util.concurrent.ConcurrentSkipListSet

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/support/reporters.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/support/reporters.kt
@@ -1,10 +1,9 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.Reporter
-import com.github.shyiko.ktlint.reporter.checkstyle.CheckStyleReporter
-import com.github.shyiko.ktlint.reporter.json.JsonReporter
-import com.github.shyiko.ktlint.reporter.plain.PlainReporter
-import me.cassiano.ktlint.reporter.html.HtmlReporter
+import com.pinterest.ktlint.core.Reporter
+import com.pinterest.ktlint.reporter.checkstyle.CheckStyleReporter
+import com.pinterest.ktlint.reporter.json.JsonReporter
+import com.pinterest.ktlint.reporter.plain.PlainReporter
 import java.io.File
 import java.io.PrintStream
 
@@ -20,7 +19,7 @@ fun reporterFor(reporterName: String, output: File): Reporter {
     return SortedThreadSafeReporterWrapper(
         when (ReporterType.valueOf(reporterName)) {
             ReporterType.checkstyle -> CheckStyleReporter(out)
-            ReporterType.html -> HtmlReporter(out)
+            ReporterType.html -> throw NotImplementedError("html reporter not yet compatible with version") // HtmlReporter(out)
             ReporterType.json -> JsonReporter(out)
             ReporterType.plain -> PlainReporter(out)
         }

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/support/ruleSets.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/support/ruleSets.kt
@@ -1,8 +1,8 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.RuleSet
-import com.github.shyiko.ktlint.core.RuleSetProvider
-import com.github.shyiko.ktlint.ruleset.experimental.ExperimentalRuleSetProvider
+import com.pinterest.ktlint.core.RuleSet
+import com.pinterest.ktlint.core.RuleSetProvider
+import com.pinterest.ktlint.ruleset.experimental.ExperimentalRuleSetProvider
 import java.util.ServiceLoader
 import kotlin.comparisons.compareBy
 

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/FormatTask.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/FormatTask.kt
@@ -1,7 +1,7 @@
 package org.jmailen.gradle.kotlinter.tasks
 
-import com.github.shyiko.ktlint.core.KtLint
-import com.github.shyiko.ktlint.core.RuleSet
+import com.pinterest.ktlint.core.KtLint
+import com.pinterest.ktlint.core.RuleSet
 import org.gradle.api.logging.LogLevel
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.OutputFile

--- a/src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/lint/LintWorkerRunnable.kt
+++ b/src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/lint/LintWorkerRunnable.kt
@@ -1,9 +1,9 @@
 package org.jmailen.gradle.kotlinter.tasks.lint
 
-import com.github.shyiko.ktlint.core.KtLint
-import com.github.shyiko.ktlint.core.LintError
-import com.github.shyiko.ktlint.core.Reporter
-import com.github.shyiko.ktlint.core.RuleSet
+import com.pinterest.ktlint.core.KtLint
+import com.pinterest.ktlint.core.LintError
+import com.pinterest.ktlint.core.Reporter
+import com.pinterest.ktlint.core.RuleSet
 import org.gradle.api.logging.Logger
 import org.jmailen.gradle.kotlinter.support.ExecutionContextRepository
 import org.jmailen.gradle.kotlinter.support.resolveRuleSets

--- a/src/test/kotlin/org/jmailen/gradle/kotlinter/support/HasErrorReporterTest.kt
+++ b/src/test/kotlin/org/jmailen/gradle/kotlinter/support/HasErrorReporterTest.kt
@@ -1,6 +1,6 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.LintError
+import com.pinterest.ktlint.core.LintError
 import org.junit.Assert.assertTrue
 import org.junit.Assert.assertFalse
 import org.junit.Before

--- a/src/test/kotlin/org/jmailen/gradle/kotlinter/support/RuleSetsTest.kt
+++ b/src/test/kotlin/org/jmailen/gradle/kotlinter/support/RuleSetsTest.kt
@@ -1,9 +1,9 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.KtLint
-import com.github.shyiko.ktlint.core.Rule
-import com.github.shyiko.ktlint.core.RuleSet
-import com.github.shyiko.ktlint.core.RuleSetProvider
+import com.pinterest.ktlint.core.KtLint
+import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.RuleSet
+import com.pinterest.ktlint.core.RuleSetProvider
 import org.jetbrains.kotlin.com.intellij.lang.ASTNode
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertTrue

--- a/src/test/kotlin/org/jmailen/gradle/kotlinter/support/SortedThreadSafeReporterWrapperTest.kt
+++ b/src/test/kotlin/org/jmailen/gradle/kotlinter/support/SortedThreadSafeReporterWrapperTest.kt
@@ -1,11 +1,11 @@
 package org.jmailen.gradle.kotlinter.support
 
-import com.github.shyiko.ktlint.core.LintError
-import com.github.shyiko.ktlint.core.Reporter
 import com.nhaarman.mockitokotlin2.inOrder
 import com.nhaarman.mockitokotlin2.mock
 import com.nhaarman.mockitokotlin2.never
 import com.nhaarman.mockitokotlin2.verify
+import com.pinterest.ktlint.core.LintError
+import com.pinterest.ktlint.core.Reporter
 
 import org.junit.Before
 import org.junit.Test

